src/sysupdate/sysupdate.c

```diff
@@ -694,6 +723,7 @@ static int context_show_version(Context *c, const char *version) {
                                           SD_JSON_BUILD_PAIR_BOOLEAN("installed", FLAGS_SET(us->flags, UPDATE_INSTALLED)),
                                           SD_JSON_BUILD_PAIR_BOOLEAN("obsolete", FLAGS_SET(us->flags, UPDATE_OBSOLETE)),
                                           SD_JSON_BUILD_PAIR_BOOLEAN("protected", FLAGS_SET(us->flags, UPDATE_PROTECTED)),
+                                          SD_JSON_BUILD_PAIR_STRV("changelog-url", changelog_urls),
```

Comment:
we nowadays try to stay clear of json field names with dashes, because this typically cannot be used as identifier in most programming languages, and we'd rather avoid to require any mapping